airbyte-cdk/python/airbyte_cdk/sources/file_based/config/file_based_stream_config.py

```diff
@@ -2,84 +2,18 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
-import codecs
-from enum import Enum
 from typing import Any, List, Mapping, Optional, Union
 
+from airbyte_cdk.sources.file_based.config.csv_format import CsvFormat
+from airbyte_cdk.sources.file_based.config.jsonl_format import JsonlFormat
+from airbyte_cdk.sources.file_based.config.parquet_format import ParquetFormat
```

Comment:
was it just for organization that we moved everything to its own file? or was there like a cyclic dependency that forced us to? this is probably better anyway but just curious

Reply:
just because the file became large